CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
  - [#1539](https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1539) Add retries for MQTT publishing with the backoffAlgorithm library.
  - [#1516](https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1516), [#1538](https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1538), [#1540](https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1540), [#1487](https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1487) Minor bugfixes and refactoring of the OTA demos and OTA PAL for POSIX.
  - [#1552](https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1552) Update broken links in demo comments and documentation.
+ - [# #1599](https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1599) In OTA demo add check before calling pthread_join.
```

Comment:
There is a new section to the CHANGELOG which documents the changes after the release. https://github.com/aws/aws-iot-device-sdk-embedded-C/blob/main/CHANGELOG.md#main
Could you move this to the above section. Also might change the wording to be consistent with other logs:
"Checking execution status of demo loop before calling pthread_join in OTA demos"

Also might want to include changes of https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1593

Reply:
Updated in https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1599/commits/5fa52c33a0217ce6ec701c5503dc61090f2ccf9c